jwk/helper.go

```diff
@@ -49,13 +49,6 @@ func AsymmetricKeypair(ctx context.Context, r InternalRegistry, g KeyGenerator,
 	if err != nil {
 		return nil, nil, err
 	}
-	keys, err := r.KeyManager().GetKeySet(ctx, set)
```

Comment:
I removed this because it served no purpose / did not make sense in this context and had no tests